templates/airflow-ec2.j2

```diff
@@ -0,0 +1,338 @@
+
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Provision EC2 instance, set up airflow on it.
+
+Parameters:
+
+  InstanceType:
+    Description: EC2 instance type
+    Type: String
+    Default: t3a.xlarge
+
+  VpcSubnet:
+    Description: Name of an existing VPC subnet to run the instance in.
+    Type: String
+
+  VpcId:
+    Description: 'ID of an existing VPC to run the instance in'
+    Type: String
+
+  EncryptVolume:
+    Type: String
+    Description: true to encrypt root volume, false (default) for no encryption
+    AllowedValues:
+      - true
+      - false
+    Default: true
+    ConstraintDescription: 'Must be true or false'
+
+  AMIId:
+    Description: The ID of the AMI to deploy
+    Type: AWS::EC2::Image::Id
+    ConstraintDescription: Valid AMIs are at https://ami.sageit.org/
+    Default: ami-0aa7d40eeae50c9a9
+
+  VolumeSize:
+    Description: The EC2 volume size (in GB)
+    Type: Number
+    Default: 20
+    MinValue: 10
+    MaxValue: 2000
+
+  BackupEc2:
+    Type: String
+    Description: true to enable daily EC2 backup, false (default) for no backup
+    AllowedValues:
+      - true
+      - false
+    Default: false
+
+  SnapshotCount:
+    Description: The number of snapshots to keep, only used if BackupEc2=true
+    Type: Number
+    MinValue: 1
+    MaxValue: 180
+    Default: 7
+    ConstraintDescription: Valid values are 1-180
+
+
+Conditions:
+  # PublicEc2Resources: !Or [!Equals [ !Ref VpcSubnet, PublicSubnet ], !Equals [ !Ref VpcSubnet, PublicSubnet1 ], !Equals [ !Ref VpcSubnet, PublicSubnet2 ] ]
+  #HasKeyName: !Not [!Equals ['', !Ref KeyName]]
+  EnableEc2Backup: !Equals [!Ref BackupEc2, true]
+
+Resources:
+
+  VpnSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: 'Allow traffic from VPN'
+      VpcId: !Ref VpcId
+      SecurityGroupIngress:
+        - CidrIp: '10.1.0.0/16'  # Sophos VPN
```

Comment:
Do we need to include both Sophos and AWS VPN?

Reply:
This I am not sure about. @BrunoGrandePhD?

Reply:
You can remove the Sophos VPN. Since Tess wrote this, I believe that it's been officially deprecated by IT.